webui_runner/plugs/wamr_app_aot/download.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+source ../../config.inc
+
+>&2 echo "wamr_app_aot download"
+mkdir -p build; cd build
+if [ ! -d wamr ]; then
+    git clone https://github.com/bytecodealliance/wasm-micro-runtime.git wamr
+fi
+cd wamr
+git clean -fd
+git submodule foreach --recursive git clean -fd
+git reset --hard
+git submodule init && git submodule update
+git submodule foreach --recursive git submodule init
+git submodule foreach --recursive git submodule update
+git pull
```

Comment:
Lines 8-17 are going to change, right?

Reply:
Yes ... it is the same for other VMs though and not specific to enabling wamr. I was working on a separate patch to do this across all git downloads consistently but can do it here now?